src/server/game/Server/WorldSocketMgr.cpp

```diff
@@ -62,7 +62,7 @@ bool WorldSocketMgr::StartWorldNetwork(Acore::Asio::IoContext& ioContext, std::s
     if (!BaseSocketMgr::StartNetwork(ioContext, bindIp, port, threadCount))
         return false;
 
-    _acceptor->AsyncAcceptWithCallback<&OnSocketAccept>();
+    _acceptor->AsyncAcceptWithCallback<OnSocketAccept>();
```

Comment:
not sure about this tbh

Reply:
So what do you suggest to do instead?

Reply:
@Kitzunu no idea.. unfortunately, I didn't have the opportunity to dig into this